stock_picking_batch_extended/__manifest__.py

```diff
@@ -1,11 +1,12 @@
 # Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
 # Copyright 2018-2020 Tecnativa - Carlos Dauden
+# Copyright 2023 FactorLibre - Boris Alias
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 {
     "name": "Stock batch picking extended",
     "summary": "Allows manage a lot of pickings in batch",
-    "version": "15.0.2.2.0",
+    "version": "16.0.1.0.0",
     "author": "Camptocamp, " "Tecnativa, " "Odoo Community Association (OCA)",
```

Comment:
Hi @BorisFL , thanks for contributing.
can you fix this old pre-commit result?
```suggestion
    "author": "Camptocamp, Tecnativa, Odoo Community Association (OCA)",
```

Reply:
I have fixed the concatenation of the authors

Reply:
thanks :)